pkg/exchange/ftx/convert.go

```diff
@@ -21,6 +22,9 @@ func toGlobalSymbol(original string) string {
 }
 
 func toLocalSymbol(original string) string {
+	if "" == symbolMap[original] {
```

Comment:
I wonder if we can use go:generate to generate a static lookup map here

Reply:
We should handle symbols without local symbols (ex. BTC-PERP)